### ISSUE TITLE
Vets-api terminal access for Tai Wilkin

### ISSUE THREAD

**Issue by TaiWilkin** [department-of-veterans-affairs/va.gov-team#79751]
### Your Name

Tai Wilkin

### Your GitHub handle (terminal access is managed through GitHub)

@TaiWilkin

### Your dsvagovcloud AWS user name (if applicable)

Tai.Wilkin

### Team, Role, and Company of the target individual

Tai Wilkin, SWE, Coforma

### Product Manager (PM) name and email

Anh Platt anh.platt@coforma.io

### Product Owner (PO) name and email

Emily Theis emily.theis@va.gov

### COR Name (leave "None" if you listed an AWS user name above)

Eunice Garcia

### Vendor Onboarding Representative (VOR) Name (leave "None" if you listed an AWS user name above)

Other - please specify in 'Additional Notes'

### Please confirm if you are listed as a team member in Atlas or are listed in Platform Team Roster

- [X] [Team Member in Atlas](https://www.va.gov/atlas/product_directory/team_members)
- [ ] [Platform Team Roster](https://vfs.atlassian.net/wiki/spaces/AP/pages/1908834623/Platform+Roster)

### Please confirm E-QIP Transmittal/Adjudication. This could be a screenshot of your Transmittal/Adjudication email, a link to another request where you've submitted this proof (i.e. AWS/SOCKS), etc.

- [X] I have attached proof that I have either transmitted my E-QIP survey or previously received a favorable decision (by including in "Additional Notes" section below)

### ArgoCD terminal access requested for the following environments

- [X] dev
- [X] staging
- [X] sandbox
- [X] production (if requesting, justify prod access in 'Additional Notes' section)

### Additional Notes

VOR - Jeff Langworthy [jeff@coforma.io](mailto:jeff@coforma.io)
<img width="997" alt="TransmissalNotice" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/21046714/964b9314-cc79-4893-a4cf-ea562fafb53b">
I need to be able to investigate and correct submission failures beyond the data present in DataDog, specifically to review the data in FormSubmissions to determine the cause of failure for claims submitted to the benefits intake API in order to rectify the failure. 

### Additional resources for Support Engineer

- [ ] For users on a VFS team but not listed as a team member in Atlas, they must start the [Platform orientation process](https://depo-platform-documentation.scrollhelp.site/getting-started/Platform-Orientation.1877344532.html)
- [ ] Instructions for handling requests [here](https://vfs.atlassian.net/wiki/spaces/PTST/pages/2183037084/Backend+Support+Responsibilities#Handle-vets-api-ArgoCD-terminal-access) - Support Only

**Comment by bdech** [comment]
@TaiWilkin  You’ve been granted access to the `dev`, `staging`, and `sandbox` environments.

Here's some additional documentation for [Vets API on EKS](https://depo-platform-documentation.scrollhelp.site/developer-docs/vets-api-on-eks#VetsAPIonEKS-TerminalAccess) that may help.

**Comment by bdech** [comment]
@TaiWilkin Access to `production` is pending OCTODE approval. We will update you as soon as we have more information.

**Comment by JonathanKamensDVA** [comment]
Approved.

**Comment by AnthonyWFlowers** [comment]
@TaiWilkin @gh_username You've been granted access to Production, 
with access limited to 6 months, expiring 10/2/24.

Please set a reminder for yourself to return to this Issue and 
request an extension within 1 week of the expiration date. You'll 
receive 1 reminder with a 72 hour window to respond. Failure to 
respond will result in your access being revoked and will require 
you to fill out a new request.

**Comment by AnthonyWFlowers** [comment]
Slack reminder [here](https://dsva.slack.com/archives/C06CY71PCAX/p1712088112601349)

**Comment by lindsay-insco** [comment]
@TaiWilkin Your Vets API Terminal Production access has expired.

Would you like to request an extension? You have 72 hours to respond, or your access will be revoked and you will need to fill out a new request.

**Comment by TaiWilkin** [comment]
Yes, I would like to request an extension. I continue to need to use the API terminal to remediate failed claims for Pensions and Burials.

**Comment by AnthonyWFlowers** [comment]
Greetings @LindseySaari @ph-One @ericboehs can one of you please give us a :thumbsup: to extend access? Thank you.

**Comment by AnthonyWFlowers** [comment]
@TaiWilkin  your Production access has been extended for 6 months, expiring 06/06/2025.

Please set a reminder for yourself to return to this Issue and request an extension within 1 week of the expiration date. You'll receive 1 reminder with a 48 hour window to respond. Failure to respond will result in your access being revoked and will require you to fill out a new request. Thank you.

**Comment by AnthonyWFlowers** [comment]
Slack reminder [here](https://dsva.slack.com/archives/C06CY71PCAX/p1733496139136169)

**Comment by GWright321** [comment]
@TaiWilkin  Your Vets API Terminal Production access has expired.

Would you like to request an extension? You have 72 hours to respond, or your access will be revoked and you will need to fill out a new request.

**Comment by TaiWilkin** [comment]
Yes, I would like to request an extension

**Comment by GWright321** [comment]
Slack Reminder [here](https://dsva.slack.com/archives/C06CY71PCAX/p1749588670028519).
